### PR TITLE
update HPA translation

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: i18n/zh/messages.zh.xlf [hunk 1]
```diff
@@ -1790,39 +1790,39 @@
...
+        <target state="new">最小副本数</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/resourcelist/horizontalpodautoscaler/template.html</context>
           <context context-type="linenumber">58</context>
         </context-group>
       </trans-unit>
       <trans-unit id="db972eb77504c0dfc832ecb36168c5b64ab6d30b" datatype="html">
         <source>Max Replicas</source>
-        <target state="new">Max Replicas</target>
+        <target state="new">最小副本数</target>
```
Comment (zehuaiWANG) [inline i18n/zh/messages.zh.xlf]: 最大副本数

Comment (hwdef) [reply i18n/zh/messages.zh.xlf]: 刚发现了，改完了。

**Comment by shu-mutou** [comment]
Don't you translate `i18n/zh-sg/messages.zh-sg.xlf`?

**Comment by hwdef** [comment]
@shu-mutou 
The current reviewer does not know about zh-sg. 
I think We temporarily link zh-sg to zh is better.
WDYT?

**Comment by shu-mutou** [comment]
@hwdef There is no particular reason to object, so create a symbolic link as shown below and submit PR.

```
cd i18n/zh-sg/
rm messages.zh-sg.xlf
ln -s ../zh/messages.zh.xlf messages.zh-sg.xlf
```

**Comment by hwdef** [comment]
@shu-mutou 
Thanks for you help.
Please review this again.

**Comment by shu-mutou** [comment]
@hwdef I'm sorry forgetting onemore thing. Could you remove `zh-cn` and `zh-sg` from `xliffmergeOptions` in package.json.

**Comment by hwdef** [comment]
@shu-mutou 
done

**Comment by shu-mutou** [comment]
/lgtm
@hwdef Thanks!

**Pull request merged**